content/maxwell1_fundamentals/dipole_sources_in_homogeneous_media/magnetic_dipole_definition/index.rst

```diff
@@ -0,0 +1,192 @@
+.. math::
+	\mathbf{m} = \mathbf{M} dV
+	:name: dip_moment_def
+
+
+From our definition of the magnetic dipole, :math:`\mathbf{m}` has units Am :math:`\!^2`.
+Each Dirac delta function carries units m :math:`\!^{-1}`, :math:`\omega` has units s :math:`\!^{-1}` and :math:`\mu` has units H/m.
+Where 1 H = 1 V :math:`\!\cdot\!` s/A, the magnetic source term (:math:`\mathbf{J_m}`) has units V/m :math:`\!^2`.
+
+
+For a magnetized rectangular block (:numref:`BarMagnet_def` left), the magnetic field outside the source region can be calculated according to Sharma (1966); a cleaner formulation can be found in Varga.
+By taking the limit as :math:`\Delta x , \, \Delta y , \, \Delta z \rightarrow dx, \, dy , \, dz`, the magnetic field generated by a magnetized rectangular block reduces to (:numref:`BarMagnet_def` right):
+
+.. math::
+	\mathbf{H_{dip}(r)} = \frac{1}{4\pi} \Bigg [ \frac{3 \mathbf{r (m \cdot r)} }{r^5} - \frac{\mathbf{m}}{r^3} \Bigg ]
+	:name: dip_field_bar
+
+
+
+
+**Current Loop Model**
+
+
+Magnetic fields are generated by the movement of electrical charges (i.e. electric current).
+Because of this, a magnetized volume in itself does not represent a physical source.
+Here, we will demonstrate how the magnetic dipole moment can be represented by an infinitessimally small loop of current.
+
+First, let us consider a large circular loop of current with radius  :math:`a` and current :math:`I` (:numref:`CurrentLoop` left).
+To obtain the primary magnetic field from the loop, we can use the :ref:`Biot-Savart law<biot_savart>`:
+
+.. math::
+	\mathbf{H (r)} = \frac{1}{4\pi} \int_C \frac{I \, d\mathbf{l} \times \mathbf{\hat r}}{r^2}
+	:name: Biot_Savart
+
+
+The analytic solution for the Biot-Savart law in this case is rather complicated and contains several elliptic integral functions; for solution see here (link).
+If the radius of the loop is much smaller than the scale of observation (:math:`a \ll r`), then the primary magnetic field due to the loop can be simplified to:
+
+.. math::
+	\mathbf{H(r)} = \frac{1}{4\pi} \Bigg [ \frac{3 \mathbf{r} (\pi a^2 I \hat n \cdot \mathbf{r)} }{r^5} - \frac{\mathbf{\pi a^2 I \hat n}}{r^3} \Bigg ]
+	:name: dip_field_loop
+
+
+where :math:`\hat n` is the unit vector normal to the area within the loop.
+The primary magnetic field for a small loop is shown in :numref:`CurrentLoop` (right).
+
+
+.. figure:: images/H_source_current_loop.png
+		:align: center
+		:figwidth: 100%
+		:name: CurrentLoop
+
+        	Magnetic field due to a loop of current. Large current loop (left). Small current loop (right). For both loops, the current is adjust such that :math:`IS` = 1 Am :math:`\!^2`.
+
+
+
+
+Notice how the primary field for a small loop is effectively identical to that of a magnetic dipole source.
+Additionally, the strength of the field depends on the product of loop's current and its area (:math:`S = \pi a^2` ).
+Therefore, if we define the dipole moment of the loop as:
+
+.. math::
+	\mathbf{m} = I \mathbf{S}
```

Comment:
Here, I am not sure if we want to be using `S`? `A` is more standard for area. Thoughts?